runbot_buildout/models/runbot_branch.py

```diff
@@ -35,3 +39,21 @@ def _compute_buildout_version(self):
             if not match:
                 continue
             this.update({'buildout_version': match.groupdict()['version']})
+
+    @api.multi
+    def get_interpreter(self):
```

Comment:
What about get interpreter from magic comment of odoo-bin file?

Reply:
that doesn't exist, repos that rely on buildouts never have any plain odoo/ocb repo in runbot, just the buildout and the code/module branches

Reply:
@moylop260 we need the interpreter to bootstrap the buildout. This bootstrap wil create the bin/buildout script that, after running the buildout, will create the odoo part that contains odoo-bin.